packages/backend/jobs-manager/service/src/modules/database/reporting/host/host-filter.model.ts

```diff
@@ -10,4 +10,5 @@ export class HostFilterModel {
   host?: Array<string>;
   firstSeenStartDate?: number;
   firstSeenEndDate?: number;
+  blocked?: 'true' | 'false';
```

Comment:
Why not use boolean?

Reply:
Because the [casting](https://github.com/typestack/class-transformer/issues/676) in the GET requests does not work as expected, it is pretty annoying and [a known issue](https://github.com/typestack/class-transformer/issues/626). Just re-googled though and could potentially use the following alternative everywhere:

```typescript
@IsBoolean()
@Transform(({ value }) => {
  if (value === 'true') return true;
  if (value === 'false') return false;
  return value;
})
```

Let me know which one you prefer.

Reply:
TBH I think I prefer the transform! At least the issue is isolated as soon as possible.